src/lib/utils.ts

```diff
@@ -183,32 +191,34 @@ export function parsePacket({ type, attributes }: ParsedEvent): Packet {
   );
   const [timeoutRevisionNumber, timeoutRevisionHeight] =
     attributesObj.packet_timeout_height?.split('-') ?? [];
-  return {
-    sequence: Long.fromNumber(attributesObj.packet_sequence ?? 0, true),
+  return Packet.fromPartial({
+    sequence: may(Long.fromString, attributesObj.packet_sequence),
```

Comment:
Do you not have to specify it's unsigned with `Long.fromString`?

Reply:
It doesn't matter here